setup.py

```diff
@@ -339,77 +367,75 @@ def lib_config_magic(prefix, libname, static_config={}):
     blas_precompiler = ('BLAS_OTHER', '1')
 print 'BLAS precompiler directive: -D%s' % blas_precompiler[0]
 
-
 # Define extension modules
 # ------------------------
 
 ext_modules = [
     Extension("horton.cext",
-        sources=get_sources('horton'),
-        depends=get_depends('horton'),
-        include_dirs=[np.get_include(), '.'],
-        extra_compile_args=['-std=c++11'],
-        language="c++"),
+              sources=get_sources('horton'),
+              depends=get_depends('horton'),
+              include_dirs=[np.get_include(), '.'],
+              extra_compile_args=['-std=c++11'],
+              language="c++"),
     Extension("horton.matrix.cext",
-        sources=get_sources('horton/matrix'),
-        depends=get_depends('horton/matrix'),
-        include_dirs=[np.get_include(), '.'],
-        extra_compile_args=['-std=c++11'],
-        language="c++"),
+              sources=get_sources('horton/matrix'),
+              depends=get_depends('horton/matrix'),
+              include_dirs=[np.get_include(), '.'],
+              extra_compile_args=['-std=c++11'],
+              language="c++"),
     Extension("horton.gbasis.cext",
-        sources=get_sources('horton/gbasis') + ['horton/moments.cpp'],
```

Comment:
This will give too long lines too easily. Another style can be used to avoid this according to PEP8:

    Extension(
        "horton.gbasis.cext",
        sources=get_sources('horton/gbasis') + ['horton/moments.cpp'],